Sources/Jetworking/Client/Client.swift

```diff
@@ -10,6 +10,8 @@ enum APIError: Error {
 public final class Client {
     public typealias RequestCompletion<ResponseType> = (HTTPURLResponse?, Result<ResponseType, Error>) -> Void
     // MARK: - Properties
+    public private(set) lazy var sessionCache: SessionCache = .init(configuration: configuration)
```

Comment:
I think `sessionCache` should be private. If there is a need to call methods on `sessionCache` it would be better to expose them via `Client` methods